### DISCUSSION TITLE
Combined Watchlist (Released) - Sorting options

### DISCUSSION THREAD

**Discussion by metaltermite** [jurialmunkey/plugin.video.themoviedb.helper#1291] [Q&A] [answered]
If I navigate to `Video Add ons > TMDb Helper > Trakt > Combined Watchlist (Released)`, the items will automatically be sorted by what *seems* like release date (descending), which is great.

However, if I add that same list as a widget, I am presented with various sorting options, none of which is release date (descending).

Is this an oversight, a limitation, or is there a way to achieve the release date sorting when adding the list as a widget?

**Comment by jurialmunkey** [comment, marked as answer]
Just an oversight. Only whitelisted the extended sorting methods for the standard watchlist not the released/anticipated versions. I'll fix the missing sort options for next release.

In the meantime -- standard watchlist using Sort by: Premiered Descending should give the same thing (but will include unaired items).
